lib/components/slideshow/fade.js

```diff
@@ -74,6 +74,7 @@ var Fade = function (_Component) {
   }, {
     key: 'componentWillUnmount',
     value: function componentWillUnmount() {
+      this.willUnmount = true;
```

Comment:
Shouldn't this be in the state of the component?

Reply:
We shouldn't use this.setState() in componentWillUnmount in React because it has no effect on code for tween.
Storing value to `this` is intentional.